src/components/globe-data-view/globe-data-view.js

```diff
@@ -0,0 +1,6 @@
+import { connect } from 'react-redux'
+import Component from './globe-data-view-component.jsx';
+
+const mapStateToProps = ({ sidebar }) => ({ isSidebarOpen: sidebar.open })
```

Comment:
if we follow the sidebars approach as described before this wouldn't have one sidebar but two `dataExplorerSidebar` and `featuredGlobeSidebar` (with better namings maybe)